api/src/pcapi/core/subscription/api.py

```diff
@@ -651,38 +653,47 @@ def get_subscription_message(user: users_models.User) -> models.SubscriptionMess
     if next_step == models.SubscriptionStep.MAINTENANCE:
         return subscription_messages.MAINTENANCE_PAGE_MESSAGE
 
-    identity_fraud_checks = sorted(
-        _get_identity_fraud_checks_for_eligibility(user, user.eligibility),
+    fraud_checks = sorted(
+        [check for check in user.beneficiaryFraudChecks if user.eligibility in check.applicable_eligibilities],
         key=lambda check: check.dateCreated,
         reverse=True,
     )
 
     dms_check = next(
         (
             check
-            for check in identity_fraud_checks
+            for check in fraud_checks
             if check.type == fraud_models.FraudCheckType.DMS and check.status != fraud_models.FraudCheckStatus.CANCELED
         ),
         None,
     )
-
     if dms_check is not None:
         return dms_subscription_api.get_dms_subscription_message(dms_check)
 
     ubble_check = next(
-        (check for check in identity_fraud_checks if check.type == fraud_models.FraudCheckType.UBBLE),
+        (check for check in fraud_checks if check.type == fraud_models.FraudCheckType.UBBLE),
         None,
     )
-
     if ubble_check is not None:
         return ubble_subscription_api.get_ubble_subscription_message(ubble_check, is_retryable=next_step != None)
 
     educonnect_check = next(
-        (check for check in identity_fraud_checks if check.type == fraud_models.FraudCheckType.EDUCONNECT),
+        (check for check in fraud_checks if check.type == fraud_models.FraudCheckType.EDUCONNECT),
         None,
     )
-
     if educonnect_check is not None:
         return educonnect_subscription_api.get_educonnect_subscription_message(educonnect_check)
 
+    user_profiling_ko_check = next(
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Comme user profiling n'est plus utilisé, est ce qu'on veut vraiment ajouter ce commit ?

Reply:
hmm il y en a toujours dans la db donc je me dis qu'on veut toujours bloquer / informer